installer/install-ntt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+version="0.4"
+os="$(uname -s | tr "[:upper:]" "[:lower:]")"
+
+case "${os}" in
+darwin|linux)
+  os="$(echo "$os" | sed -e "s/\b\(.\)/\u\1/g")"
+  url="https://github.com/nokia/ntt/releases/download/v${version}/ntt_${os}_x86_64.tar.gz"
```

Comment:
use `"${os^}"`

Reply:
This shell expansion is not supported on Darwin platforms, because they only provide bash versions 3.x